drivers/interrupt_controller/Kconfig.plic

```diff
@@ -7,17 +7,50 @@ config PLIC
 	depends on DT_HAS_SIFIVE_PLIC_1_0_0_ENABLED
 	select MULTI_LEVEL_INTERRUPTS
 	select 2ND_LEVEL_INTERRUPTS
+	select IRQ_HAS_AFFINITY
 	help
 	  Platform Level Interrupt Controller provides support
 	  for external interrupt lines defined by the RISC-V SoC.
 
 if PLIC
 
+config PLIC_IRQ_AFFINITY
+	bool
+	default y if IRQ_AFFINITY
+	depends on SMP
+	depends on MP_MAX_NUM_CPUS > 1
+	help
+	  Internal helper to indicate that IRQ affinity is enabled.
+
+config IRQ_AFFINITY_MASK
```

Comment:
Is it expected that we have two `config IRQ_AFFINITY_MASK` entries? (first in arch/Kconfig)

Reply:
Yes, an implementation can have a default value by overridding the one in `arch/Kconfig`